cmd/bnsd/scenarios/gov_test.go

```diff
@@ -0,0 +1,130 @@
+package scenarios
+
+import (
+	"testing"
+	"time"
+
+	"github.com/iov-one/weave"
+	bnsdApp "github.com/iov-one/weave/cmd/bnsd/app"
+	"github.com/iov-one/weave/cmd/bnsd/client"
+	"github.com/iov-one/weave/cmd/bnsd/scenarios/bnsdtest"
+	"github.com/iov-one/weave/coin"
+	"github.com/iov-one/weave/weavetest"
+	"github.com/iov-one/weave/x/cash"
+	"github.com/iov-one/weave/x/gov"
+)
+
+func TestGovProposalCreateAndExecute(t *testing.T) {
+	var (
+		alice = client.GenPrivateKey()
+		bobby = client.GenPrivateKey()
+		carl  = client.GenPrivateKey()
+	)
+
+	env, cleanup := bnsdtest.StartBnsd(t,
+		bnsdtest.WithMinFee(coin.NewCoin(0, 0, "IOV")),
+		bnsdtest.WithAntiSpamFee(coin.NewCoin(0, 0, "IOV")),
+		bnsdtest.WithGovernance(
+			weave.AsUnixDuration(120*time.Second),
+			[]weave.Address{
+				alice.PublicKey().Address(),
+				bobby.PublicKey().Address(),
+				carl.PublicKey().Address(),
+			}),
+	)
+	defer cleanup()
+
+	// Alice needs funds because a successful proposal execution will
+	// transfer coins from her account into Carls.
+	bnsdtest.SeedAccountWithTokens(t, env, alice.PublicKey().Address())
```

Comment:
This will fail. Governance has no access to this.

You can fill up the coins in the *governance* address. Then those could be sent.

You could also try an update electorate msg, which should be done if admin was set properly